sonar-ws/src/main/java/org/sonarqube/ws/client/issue/IssueFilterParameters.java

```diff
@@ -47,7 +47,7 @@
   public static final String FILE_UUIDS = "fileUuids";
   public static final String ON_COMPONENT_ONLY = "onComponentOnly";
   public static final String RULES = "rules";
-  public static final String ACTION_PLANS = "actionPlans";
+  public static final String DREPRECATED_ACTION_PLANS = "actionPlans";
```

Comment:
This constant should be marked as deprecated since 5.5 (even if it's not an API, it's used in the WS which is an API, and it will help us to know when we can remove it)